### ISSUE TITLE
Export Issues

### ISSUE THREAD

**Issue by TimoFriedri** [mikedh/trimesh#144]
Hi,

I wanted to export a mesh using a filename for file_obj, which should work according to the function description.
I tried to export as "filename.obj"

In line 37 of trimesh/io/export.py you open a file in byte mode, but later the export object to be written is a string and causes the error.

I just changed it to 'b' from 'wb' for me for now. I don't know if this affects other export modes.

Sorry, I have no time ATM for a PR and testing but I wanted to communicate this bug.
You should consider to add unit tests for filename export mode.

Cheers,

Timo

**Comment by mikedh** [comment]
Thanks for the report! I was not able to reproduce this, I just added a test which does export/reimport by name in https://github.com/mikedh/trimesh/commit/f78d99aaeac04f93051c39ba8c0de36745f4eb24 but it passed on Python2/3 in both Windows and Linux for me. 

This issue should be handled by [magic in util](https://github.com/mikedh/trimesh/blob/master/trimesh/util.py#L1736-L1762) that was added a few months ago- are you on a recent trimesh (`pip install -U trimesh`)? What platform/python version are you running?
